src/studio/nuxeo-studio/studio-overview-and-concepts.md

```diff
@@ -161,10 +161,11 @@ When creating a new application, the usual configuration steps are:
 <!--Nuxeo Studio Designer and parts of Modeler use the recent W3C standard Web Components and Google Polymer framework (legacy mode). The [Google Polymer framework](https://www.polymer-project.org/) comes with [polyfills for Web Components](https://www.webcomponents.org/polyfills), a library that extends the support of web components standard to more browsers.
 Nevertheless, due to browser support to some shadow DOM features and third-party libraries used in Nuxeo Studio, some browsers are not supported.-->
 
-Nuxeo Studio supports the following browsers:
+Nuxeo Studio supports the latest stable version of the following browsers:
 {{! multiexcerpt name='webui-supported-browsers'}}
  - Google Chrome
```

Comment:
We need a link like "Please refer top the Current Chrome version release notes section of https://support.google.com/chrome/a/answer/7679408?hl=en"

Reply:
I don't see the point

Reply:
Ah, I think I understand: to check if people are using the latest version...

Reply:
So if we want to do that, I believe it doesn't make sense since we're not telling people how to check  what their current version is. Pushed a new commit to add a paragraph around that explicitly.

Reply:
👍